### ISSUE TITLE
"abstract-method" warning should not appear for abstract classes

### ISSUE THREAD

**Issue by Ronkiro** [PyCQA/pylint#3310]
I am not sure if i am following the correct pattern, but i think i shouldn't receive any warnings for this approach.

### Steps to reproduce
1. Declare a "pure abstract" class:
```
in a file:
class A(object, metaclass=ABCMeta):
    @abstractmethod
    def mymethod(self):
        pass

    @abstractmethod
    def myothermethod(self):
        pass
```
2. Declare another class, that is a subclass from the before's one.
```
In other file:
class B(A):
    def myothermethod(self):
        # Do something
```
You can also declare B as ABC.
3. The class B now throws a "Method mymethod is abstract in class 'A' but not overridden"

### Current behavior
"Method mymethod is abstract in class 'A' but not overridden"

### Expected behavior
I think that this warning shouldn't appear, as the purpose isn't to override.

### pylint --version output
```
pylint 2.4.4
astroid 2.3.3
Python 3.6.0 (v3.6.0:41df79263a11, Dec 23 2016, 08:06:12) [MSC v.1900 64 bit (AMD64)]
```

**Comment by NeilGirdhar** [comment]
Just came here looking for this issue.  While I agree with you, this is a duplicate of my issue https://github.com/PyCQA/pylint/issues/843 and https://github.com/PyCQA/pylint/issues/565.

**Comment by Ronkiro** [comment]
Thanks for commenting out @NeilGirdhar, i will be closing this issue in favor of #565.
